controllers/ycsbbench/job.go

```diff
@@ -0,0 +1,58 @@
+package ycsbbench
+
+import (
+	"fmt"
+	perfv1alpha1 "github.com/xridge/kubestone/api/v1alpha1"
+	"github.com/xridge/kubestone/pkg/k8s"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strconv"
+)
+
+func formatArgs(cr *perfv1alpha1.YcsbBench) []string {
+	args := []string{
+		cr.Spec.Database,
+		"-P", fmt.Sprintf("workloads/workload%s", cr.Spec.Workload),
+	}
+
+	if cr.Spec.Options != (perfv1alpha1.YcsbBenchOptions{}) {
+		if cr.Spec.Options.Threadcount > 0 {
```

Comment:
From the API Spec one could expect that all the YCSB parameters are supported, but currently only Threadcount and Target will be taken care of. 
I'm wondering if it would be possible to make a generic solution, where we'd map the option directly to the exposed API?

Reply:
Ignore this please: You are exposing a struct, not a generic map. Sorry, my morning coffee did not kick in just yet.  😴